Pages/Index.cshtml.cs

```diff
@@ -8,64 +8,64 @@
 namespace DotnetBunnyLogBrowser.Pages
 {
     public class IndexModel : PageModel
-	{
-		public List<BunnyJob> GetJobs()
-		{
-            var job_names=Directory.GetDirectories(JobsConfig.Get().JobsDirectory, JobsConfig.Get().JobsPrefix+"*");
-			var jobs=new List<BunnyJob>(job_names.Length);
-			var client=new WebClient();
+    {
+        private static string PathToName(string path, char slash = '/')

```

Comment:
I would use `Path.DirectorySeparatorChar` instead of hardcoding the value of `slash`. That should make this program more portable on non-*nix platforms.